ribasim_qgis/scripts/run_qgis_tests.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
+
+qgis_process = subprocess.run(
+    [
+        "qgis",
+        "--profiles-path",
+        ".pixi/qgis_env",
+        "--version-migration",
+        "--nologo",
+        "--code",
+        "ribasim_qgis/scripts/qgis_testrunner.py",
+        "ribasim_qgis.tests",
+    ],
+    stdout=subprocess.PIPE,
+    stderr=subprocess.STDOUT,
+    text=True,
+)
+
+print(qgis_process.stdout)
+qgis_process.check_returncode()
+if any(s in qgis_process.stdout for s in ["QGIS died on signal", "FAILED"]):
```

Comment:
Add why we need this check.

Because qgis always returns 0 when all tests have run, even when the tests failed.